libpod/options.go

```diff
@@ -1024,8 +1024,16 @@ func WithCgroupNSFrom(nsCtr *Container) CtrCreateOption {
 			return err
 		}
 
-		ctr.config.CgroupNsCtr = nsCtr.ID()
+		for _, ns := range nsCtr.config.Spec.Linux.Namespaces {
```

Comment:
this might result in some nil pointers, no?

Reply:
Yup, is this not checked elsewhere?

Reply:
Thank you for the comments.
I will use `Container.hasNamespace()`, which checks nil pointers:
  https://github.com/containers/podman/blob/main/libpod/container_internal.go#L2321